tests/test_fmi_master.py

```diff
@@ -206,6 +207,59 @@ def test_basic_simulation_txt_file_naming_exists(self):
         
         assert os.path.isfile("Test1.txt"), "Test1.txt does not exists"
         assert os.path.isfile("Test2.txt"), "Test2.txt does not exists"
+    
+    @testattr(stddist = True)
+    def test_basic_simulation_csv_file_naming_exists(self):
+        models, connections = self._load_basic_simulation()
+        
+        opts = {"result_handling":"csv", "result_file_name": {models[0]: "Test1.csv", models[1]: "Test2.csv"}}
+        
+        res = self._sim_basic_simulation(models, connections, opts)
+        
+        assert os.path.isfile("Test1.csv"), "Test1.csv does not exists"
+        assert os.path.isfile("Test2.csv"), "Test2.csv does not exists"
+    
```

Comment:
"result_handling": "memory" is also valid now, possible add a test for that as well?

Reply:
Fixed.